cmd/kubebuilder/create/config/config.go

```diff
@@ -75,4 +75,5 @@ func AddCreateConfig(cmd *cobra.Command) {
 	configCmd.Flags().StringVar(&controllerImage, "controller-image", "", "name of the controller container to run.")
 	configCmd.Flags().StringVar(&name, "name", "", "name of the installation.  used to generate the namespace and resource names.")
 	configCmd.Flags().StringVar(&output, "output", filepath.Join("hack", "install.yaml"), "location to write yaml to")
+	configCmd.Flags().BoolVar(&skipmap, "skip-map", true, "if set to true, skip validation of map in CRD")
```

Comment:
May be rename the flag to "skip-map-validation"

Also, text can be more explicit:
`if set to true, skip generating validation schema for map type` ?

Reply:
Sounds good. I will revise the description.

Reply:
Done